client/app/visualizations/chart/index.js

```diff
@@ -311,6 +311,7 @@ export default function init(ngModule) {
     registerVisualization({
       type: 'CHART',
       name: 'Chart',
+      isDefault: true,
```

Comment:
This is to make sure Chart is always used first?

Reply:
Yes. For some reason Word Cloud became first and tests started failing. I think it's useful anyway.

Reply:
I think the same happened in the details viz pull request. In the future it's better to push such changes in a separate PR as they are easier to merge and useful beyond this one (no need to do this now though).

Reply:
NP, I'll pick this fix to another PR

Reply:
@arikfr getredash/redash#3944